examples/complianceApplications.js

```diff
@@ -0,0 +1,45 @@
+var Plivo =  require('../dist/rest/client.js');
+var client = new Plivo.Client();
+
```

Comment:
All looks very good. Just check if we need to keep console.log in master as well.

Reply:
Those are a chain of promises when someone runs any example file. 
To differentiate between each call